src/io/leopart_io.h

```diff
@@ -0,0 +1,16 @@
+// Copyright: (c) 2020 Chris Richardson and Jakob Maljaars
+// This file is part of LEoPart-X, a particle-in-cell package for DOLFIN-X
+// License: GNU Lesser GPL version 3 or any later version
+// SPDX-License-Identifier:    LGPL-3.0-or-later
+
+#pragma once
+
+#include "../Particles.h"
```

Comment:
Should avoid having "../" in the code. Better to use `<Particles.h>` and make sure that the path is in the include path.

Reply:
Deleted

Reply:
looks good